packages/nextjs/app/(authenticated)/home/page.tsx

```diff
@@ -1,13 +1,40 @@
+"use client"
+import { formatLocalTime, truncateAddress } from "~~/utils/format";
 import { classes } from "./page.styles";
+import { useAccount, useConnect } from "@starknet-react/core";
+import { useEffect, useState } from "react";
+
 
 const Home = () => {
+  const { address } = useAccount();
+  const {connector } = useConnect(); 
+  const [currentTime, setCurrentTime] = useState<string>("");
```

Comment:
```suggestion
const [currentTime, setCurrentTime] = useState<string>(formatLocalTime(now));
```

Lets try that to avoid the flikering of the date on first load